outdated/modules/httpd.reb

```diff
@@ -38,12 +53,14 @@ sys/make-scheme [
 
     spec: make system/standard/port-spec-head [port-id: actions: _]
 
-    wake-client: func [event [event!] /local client request response this][
+    wake-client: function [event [event!]] [
         client: event/port
 
         switch event/type [
-            read [
```

Comment:
Non-stylistic change: SWITCH is now evaluative, so it's **switch 'read ['read [...]]** and not **switch 'read [read [...]]**.

For the moment, there are still warnings if you try to use WORD!s in evaluative slots that are not datatypes.  So you can say **switch type of 3 [integer! [...]]** and that works, but not **x: 3 | switch 3 [x [...]]**.  You can work around this by putting other things in GROUP!s.  Exceptions are: `default`, `elide`, and `comment`.